pages/landing-page.js

```diff
@@ -30,66 +32,60 @@ export default function LandingPage() {
     </>
   );
 }
-
-const ParagraphContainer = styled.div`
-  margin: 50px 30px 0px 30px;
-  padding: 60px 40px;
-  background-color: rgba(255, 255, 236, 1);
-  border: 2px solid black;
-  box-shadow: 2px 2px 1px rgba(0, 0, 0, 0.2), -2px -2px 1px rgba(0, 0, 0, 0.2);
+const AppContainer = styled.div`
+  height: 100vh;
+  bottom: 60rem;
```

Comment:
As far as I can see, the `bottom` property does not do anything here:
"The bottom [CSS](https://developer.mozilla.org/en-US/docs/Web/CSS) property participates in setting the vertical position of a [positioned element](https://developer.mozilla.org/en-US/docs/Web/CSS/position). It has no effect on non-positioned elements." [(see on MDN)](https://developer.mozilla.org/en-US/docs/Web/CSS/bottom)
Besides, why would you do that?